examples/async_pipeline.ipynb

```diff
@@ -0,0 +1,564 @@
+{
```

Comment:
<p>"By default,  the<code>Pipeline</code>class in Haystack is a regularly Python class"</p><p> </p><p>"a" missing or maybe transform the adverb into an adjectiv?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepset-ai/haystack-experimental/pull/142/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>